src/diet/index.ts

```diff
@@ -61,6 +61,10 @@ function expectedAdventures(
     (itemType(item) === "booze" && item.notes?.includes("BEER"))
       ? 1.5
       : 1.3;
+  const seasoningAdventures =
+    item.adventures.split(/[-–—]/)[1] - item.adventures.split(/[-–—]/)[0] <= 1
```

Comment:
What happens to items that do not have one of those three characters in the adventures field?
Why split based on those three characters, isn't a normal hyphen sufficient `-`?

Reply:
I based the characters from earlier in the file here:

```
 if (item.adventures === "") return 0;
  const [min, recordedMax] = item.adventures
    .split(/[-–—]/)
    .map((s) => parseInt(s));
```

Bu actually I can just use those values and not split again, so made that change

Reply:
I think you want `max` because `recordedMax` can be undefined.

Reply:
Yeah haha, got a little ahead of myself, already changed it, thanks!